general/config.cpp

```diff
@@ -35,6 +35,26 @@ void config::CudaDeviceSetup(const int device)
 #endif
 }
 
+// *****************************************************************************
+// * RajaDeviceSetup will set: gpu_count, dev, cuDevice, cuContext & cuStream
+// *****************************************************************************
+void config::RajaDeviceSetup(const int device)
+{
+#ifdef __NVCC__
+   cudaGetDeviceCount(&ngpu);
+   MFEM_ASSERT(ngpu>0, "No CUDA device found!");
+   cuInit(0);
+   dev = device;
+   cuDeviceGet(&cuDevice,dev);
+   cuCtxCreate(&cuContext, CU_CTX_SCHED_AUTO, cuDevice);
+   cuStream = new CUstream;
```

Comment:
Is cuStream ever deleted?

Reply:
Good catch, I just added a destructor for the config class.